src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityProcessingArray.java

```diff
@@ -274,11 +275,23 @@ protected Recipe findRecipe(long maxVoltage, IItemHandlerModifiable inputs, IMul
         }
 
         @Override
-        public boolean trimOutputs() {
+        public Pair<Boolean, Integer> trimItemOutputs() {
             MetaTileEntity mte = MachineItemBlock.getMetaTileEntity(currentMachineStack);
 
```

Comment:
could these checks for macerators be made generic to something like this ?
            if (mte.getExportItems().getSlots() < mte.getRecipeMap().getMaxOutputs()) {
                return Pair.of(true, mte.getExportItems().getSlots());
            }